.github/dependabot.yml

```diff
@@ -4,7 +4,7 @@ updates:
   directory: /
   schedule:
     interval: daily
-- package-ecosystem: pip
```

Comment:
do we not still need this ?

Reply:
There is no docs folder in the project so dependabot doesn’t identify any dependency to update.